pkg/arvo/lib/hood/drum.hoon

```diff
@@ -396,7 +396,7 @@
            =?  +>.$  !=(%hush lev)
              (se-text "crud: %{(trip p.told)} event failed")
            ?.  =(%loud lev)  +>.$
-           (se-dump q.told)
+           (se-dump (flop q.told))
```

Comment:
This looks wrong -- the `%talk` case of `$told` is documented to print "in order", while `$tang` prints in reverse order.

/cc @Fang-

Reply:
I may have a different solution...